backend/lib/edgehog_web/schema/update_campaigns_types.ex

```diff
@@ -150,6 +150,20 @@ defmodule EdgehogWeb.Schema.UpdateCampaignsTypes do
     field :push, non_null(:push_rollout_input)
   end
 
+  @desc """
+  Properties that can be updated for a Push Rollout Mechanism
+  """
+  input_object :push_rollout_update do
+    @desc """
+    The maximum number of in progress updates. The Update Campaign will have \
+    at most this number of OTA Operations that are started but not yet \
+    finished (either successfully or not).
+
+    Must be greater than (or equal to) the current value.
+    """
+    field :max_in_progress_updates, :integer
```

Comment:
Should we also make `:max_in_progress_updates` non nullable?
```suggestion
    field :max_in_progress_updates, non_null(:integer)
```